internal/runners/activate/namespace.go

```diff
@@ -51,12 +54,6 @@ func (r *NamespaceSelect) Run(namespace string, preferredPath string) (string, e
 		return "", err
 	}
 
-	// Save path for future use
```

Comment:
The project mapping is added whenever we source a project file, therefore also during the initial activation. And that suffices.  I tested that

```
state activate ActiveState/Perl-5.32 --path /tmp/a-new-location
state projects
```

shows the new location of Perl-5.32.